src/modules/i18n/containers/__tests__/I18nProviderContainer.spec.js

```diff
@@ -13,78 +13,84 @@ jest.mock('../../../i18n/LanguageDetector')
 
 const mockStore = configureMockStore()
 
+// jest.resetModules() produces a invariant violation: Invalid hook call https://github.com/facebook/jest/issues/8987
```

Comment:
Is this also required although you downgraded jest again?

Reply:
Yes, sadly still required